Workade/Views&ViewModels/Home/CollectionView/MagazineCollectionViewCell.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  MegazineCollectionViewCell.swift
+//  Workade
+//
+//  Created by Hyeonsoo Kim on 2022/10/22.
+//
+
+import UIKit
+
+class MagazineCollectionViewCell: UICollectionViewCell {
+    var magazineId: Int?
+    
+    private lazy var backgroundImageView: CellImageView = {
+        let imageView = CellImageView(bounds: bounds)
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return imageView
+    }()
+    
+    private lazy var bookmarkButton: UIButton = {
+        let button = UIButton()
+        button.setImage(SFSymbol.bookmark.image, for: .normal)
+        button.addTarget(self, action: #selector(bookmarkButtonTapped), for: .touchUpInside)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        
+        return button
+    }()
+    
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 2
+        label.font = .systemFont(ofSize: 15, weight: .semibold)
```

Comment:
```suggestion
        label.font = .customFont(for: .subHeadline)
```
[P2] subHeadline이 Bold체인데 코드 상 디자인 시스템에는 regular로 잡혀있었네요.. 죄송합니다ㅠㅠ
일단은 디자인 시스템 코드로 가주시면 다음 PR때 폰트 익스텐션에서 제가 수정하도록 하겠습니다.

Reply:
아아 괜찮습니당 ㅎ 그 figma랑 최대한 똑같이 맞추려다보니 임시로 systemFont를 쓰긴했는데 custom으로 수정해놓겠습니다!